### PR TITLE
Add new sidekiq service files

### PR DESCRIPTION
**Pull request by dhollinger**

### CODE REVIEW

**Review by bastelfreak — APPROVED** [review]

File: resources/puppet-webhook.service [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 
 [Service]
-EnvironmentFile=-/etc/sysconfig/puppet-webhook
-EnvironmentFile=-/etc/default/puppet-webhook
-WorkingDirectory=/opt/voxpupuli/webhook
-ExecStart=/opt/voxpupuli/webhook/bin/bundle exec puma config.ru
-ExecReload=/bin/kill -HUP $PID
-KillMode=process
+Type=oneshot
+ExecStart=/bin/true
```
Comment (igalic) [inline resources/puppet-webhook.service:7]: i am confused about what exactly is being one-shot started here

**Review by igalic — COMMENTED** [review]
🙋🏻‍♀️

File: resources/puppet-webhook.service (hunk 1 above)
Comment (bastelfreak) [reply resources/puppet-webhook.service:7]: no process. but this unit triggers puppet-webhook-sidekiq and puppet-webhook-app

Comment (igalic) [reply resources/puppet-webhook.service:7]: ah, by those others being PartOf, now i see

**Pull request merged**